src/content/reference/react/index.md

```diff
@@ -97,25 +94,25 @@ function TodoList({ todos, tab, theme }) {
 }
 ```
 
-Sometimes, you can't skip re-rendering because the screen actually needs to update. In that case, you can improve performance by separating blocking updates that must be synchronous (like typing into an input) from non-blocking updates which don't need to block the user interface (like updating a chart).
+Terkadang, Anda tidak dapat melewatkan *rendering* ulang karena layar sebenarnya perlu pembaruan. Di dalam kasus itu, Anda dapat meningkatkan performa dengan memisahkan memblokir pembaruan yang harus sinkron (seperti mengetik masukan) dari pembaruan non-pemblokiran yang tidak perlu memblokir antarmuka pengguna (seperti memperbarui bagan).
```

Comment:
```suggestion
Terkadang, Anda tidak dapat melewatkan pe-*render*-an ulang karena layar sebenarnya perlu pembaruan. Di dalam kasus itu, Anda dapat meningkatkan kinerja dengan memisahkan pembaruan penghalang yang harus sinkron (seperti mengetik masukan) dari pembaruan yang bukan penghalang yang tidak perlu memblokir antarmuka pengguna (seperti memperbarui bagan).
```

Reply:
Shall we add "blocking" as "*penghalang*" into the glossary, @resir014 @r17x @mhaidarhanif @mazipan?